src/walkway.js

```diff
@@ -324,8 +328,10 @@
   function getPolylineLength(polyline) {
     var dist = 0;
     var x1, x2, y1, y2;
+    var i;
```

Comment:
Agree with everything Marco mentioned above. This code was merged as a pr and I should have been more diligent here. :+1: for all these tweaks.